src/components/Navigation/Footer.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import withStyles from 'isomorphic-style-loader/lib/withStyles';
+import Grid from 'react-bootstrap/lib/Grid';
+import Row from 'react-bootstrap/lib/Row';
+
+import styles from './Footer.scss';
+
+const Footer = React.createClass({
+
+  render() {
+    const url = [
+      'https://github.com/kodeklubben/oppgaver/',
+      'http://kidsakoder.no/',
+      'https://github.com/kodeklubben/kodeklubben.github.io/archive/master.zip',
+      'http://kidsakoder.no/2015/07/03/kodeklubben-trondheim-utvikler-materiell-i-sommer/'
+    ];
+
+    return (
+      <Grid fluid={true} className={styles.container}>
+        <Row className={styles.center}>
+          <p>Bidra?</p>
```

Comment:
Nei, men det er ikke bare barn og lærere/veiledere som bruker siden, unge og voksne som vil lære seg programmering og andre som ikke nødvendigvis kommer innenfor lærer/veileder kategorien er også en brukere. Tenker f.eks. hvis noen er innom siden for å sjekke den ut, og ikke finner linken for å bidra siden den bare vises i en bestemt modus. Men hvis vi designer primært med hensyn på bruk i skolen er jeg enig med deg. Legger til en hide-funksjon, så ser vi om vi ønsker å beholde den eller ikke, fort gjort å endre på uansett :smiley: